src/kemalyst-model.cr

```diff
@@ -235,7 +235,7 @@ class Kemalyst::Model
     end
   end # End of Fields Macro
 
-  def set_attributes(args : Hash(Symbol | String, String))
+  def set_attributes(args : Hash(Symbol | String, String | Int32 | Float32))
```

Comment:
All three of those have `to_i, to_f, and to_s`. I added them to make it easier to pass params to them. In some cases you might want to pass and id or count or some other number as a hard coded value. It will make it a lot easier if you don't have to wrap them in quotes. Turns out this will cause the Time parse to mess up but so will strings without the correct formatting. We can easily just tell time to convert it to string first but we'll also have to do checks to make sure it's a parseable string. We should probably add some sort of working time fields to the generator too.